src/app/components/item/item.component.html

```diff
@@ -3,9 +3,12 @@
         <i class="material-icons" id="play" *ngIf="track.isPlaying">pause_circle_outline</i>
         <i class="material-icons" id="pause" *ngIf="!track.isPlaying">play_circle_outline</i>
     </button>
-    <h6 [style.color]="track.isPlaying ? '#00d949' : '#ffffff'">{{track.name}}</h6>
+    <div>
+        <div><h6 [style.color]="track.isPlaying ? '#00d949' : '#ffffff'">{{track.name}}</h6></div>
```

Comment:
you should do this through the scss. Imagine that the company decided to change the primary color of the application - what should you do? It would be better and simpler to use scss variables. In this way, you will change the color value only in one place of code.